README.md

```diff
@@ -1,46 +1,429 @@
+### Create a new DAO with schemes
+
+Create a new DAO with schemes by including a "schemes" array.  This will register the schemes with all default permissions and voting machines. See `SchemesConfig` and `SchemeConfig`.
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  schemes: [
+    { name: "SchemeRegistrar" },
+    { name: "UpgradeScheme" },
+    { name: "GlobalConstraintRegistrar" }
+  ]
+});
+```
+
+### Create a new DAO overriding the default voting machine
+
+Override the default voting machine by adding a "votingMachines" element.  This will override the default voting machine, either at the root level to apply to all schemes, or within each scheme element to override for the specific scheme. See `NewDaoVotingMachineConfig` and `SchemesConfig`.
+
+#### Default root-level override for all schemes
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  votingMachineParams: {
+    votePerc: 45,
+    ownerVote:false
+  }
+});
 ```
 
-You can also override the default configuration settings with environment variables:
+#### Scheme-specific override
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  schemes: [
+    { name: "SchemeRegistrar" },
+    { name: "UpgradeScheme" },
+    {
+      name: "GlobalConstraintRegistrar",
+      votingMachineParams: {
+        votePerc: 30,
+      }
+    }
+  ]
+});
+```
+
+### Create a new DAO with a non-Universal Controller
+
+Create a DAO using the DAOstack `Controller` contract by passing `false` for `universalController`.
+
 ```javascript
-gasLimit=7000000 node yourprojectindex.js
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  universalController: false
+});
 ```
 
+### Create a new DAO with a non-default DaoCreator scheme
+
+Create a DAO using an alternative `DaoCreator` scheme by passing its address for `daoCreatorScheme`.  The alternative scheme must implement the same interface as the Arc `DaoCreator` contract.
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  daoCreatorScheme: anAddress
+});
+```
+
+### Get a previously-created DAO
+
+Get a previously-created DAO using the avatar address:
+
+```javascript
+const dao = await DAO.at(daoAvatarAddress);
+```
+
+### Get the DAOstack Genesis DAO
+
+The DAOstack DAO is named "Genesis".  Obtain it like this:
+
+```javascript
+const genesisDao = await DAO.getGenesisDao();
+```
+
+## Using the Arc Contracts
+
+Arc.js wraps every Arc contract in a JavaScript "wrapper" class.  Every wrapper class inherits from `ExtendTruffleContract` which provides a common set of functions in addition to all of the functions implemented by the specific Arc contract being wrapped.
+
+### Categories of Arc Contracts
```

Comment:
Replace this section with a link to arc/docs
doing that is more clean and will prevent from  update arc.js readme when ever there is an update on arc docs .

Reply:
I have responded more than once elsewhere in the PR about splitting things out.  (Here for example)[https://github.com/daostack/arc.js/pull/74#issuecomment-364263579].  That task needs to be a separate PR.

Reply:
Hmmm, but I think I misunderstood your request, hang on...

Reply:
I just added a reference to the Arc documentation, here:   https://github.com/daostack/arc.js/tree/documentationtake2#daostack-arcjs